src/components/Auth/DeleteAccount/DeleteAccount.react.js

```diff
@@ -219,7 +220,7 @@ class DeleteAccount extends Component {
           <StaticAppBar />
         </div>
         <div style={body}>
-          <Paper style={style} zDepth={5}>
+          <Paper style={style} className="delete-account" zDepth={5}>
```

Comment:
Don't use css, use the style object created for the component.

Reply:
@AakashMallik the css is used for responsiveness of the page. media queries are the best ways for the same. That's why I went for it.